src/features/dashboard/dashboardModals/notificationModal.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react'
+import cn from 'classnames'
+import moment from 'moment'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import Notification from '../../../common/components/Notification'
+import Scrollbar from '../../../common/components/Scrollbar'
+import Radio from '../components/Radio'
+import Select, { TOption } from '../../../common/components/Select/Select'
+import DatePicker from '../../../common/components/DatePicker'
+import { SelectButton } from '../../../common/components/Buttons'
+
+export type TNotification = {
+  id: number
+  type: string
+  title: string
+  status: 'read' | 'unread'
+}
+
+export type TNotificationModal = {
+  notifications: TNotification[]
+  isOpen: boolean
+  handleClose?: () => void
+}
+
+const TTypesOptions = [
+  { value: 'psl', label: 'PSL' },
+  { value: 'psl 1', label: 'PSL 1' },
+  { value: 'psl 2', label: 'PSL 2' },
+  { value: 'psl 3', label: 'PSL 3' },
+]
+
+const dateFormat = 'DD/MM/YYYY'
+
+const dateButtons = [
+  { title: 'Today' },
+  { title: 'Yesterday' },
+  { title: 'Last 7 days' },
+  { title: 'Last 30 days' },
+  { title: 'Last 3 month' },
+  { title: 'Last 12 month' },
+]
+
+const NotificationModal = ({
+  notifications,
+  isOpen,
+  handleClose,
+}: TNotificationModal): JSX.Element => {
+  const [filter, setFilter] = useState<string>('all')
+  const [type, setType] = useState<TOption | null>(TTypesOptions[0])
+  const [startDate, setStartDate] = useState<Date | null>(new Date())
+  const [endDate, setEndDate] = useState<Date | null>(null)
+  const [dateTitle, setDateTitle] = useState<string>('Today')
+
+  const handleFilter = (event: React.ChangeEvent<HTMLInputElement>): void =>
+    setFilter(event.target.value)
+
+  const handleSelect = (option: TOption | null): void => setType(option)
+
+  const handleDate = (dates: [Date, Date] | null): void => {
+    if (dates) {
+      const [start, end] = dates
+      setStartDate(start)
+      setEndDate(end)
+      if (start && !end) {
+        setDateTitle(moment(start).format(dateFormat))
+      } else if (start && end) {
+        setDateTitle(
+          `${moment(start).format(dateFormat)} - ${moment(end).format(
```

Comment:
Please use datejs instead of moment, at this place and other places, as moment is deprecated and we will remove it someday.
In addition, we have `dateformat` library for the same purpose.